game/src/main/scala/io/aigar/game/serializable/GameState.scala

```diff
@@ -18,6 +18,7 @@ case class Dimensions(
 )
 case class Cell(
   id: Int,
+  radius: Int,
   mass: Int,
```

Comment:
Can we remove the mass? I don't think we'll need it anymore (we only use `total_mass`). We can re-add it later if we want to.